app/controllers/api/v1/jwt_api/protocol_subscriptions_controller.rb

```diff
@@ -8,11 +8,6 @@ def mine
           render json: current_user.my_protocols(true),
                  each_serializer: Api::ProtocolSubscriptionSerializer
         end
-
-        def delegated_protocol_subscriptions
```

Comment:
we don't want this? Like a list of all protocolsubscriptions you have running for others? (I'm thinking Mentordashboard in react kindof stuff)

Reply:
Well the thing is that this was only used for the SDV platform, and now the SDV platform doesn't use it anymore. So when we do make a route that will be used, I'm in favor of adding it back, but I'm not in favor of adding methods right now just because we think they might be used later because what often happens is when that time arrives we'll have forgotten all about this method and just write a new one and then we have another chunk of unused code.